docs/platform/howto/change-your-email-address.rst

```diff
@@ -1,23 +1,17 @@
 Change your email address
-=========================
+==========================
 
-It is not currently possible to change your login email address directly. It is possible, however, to migrate your Aiven resources to another email address.
+You can't change your login email address directly. Instead, you can invite the new email address to your organization and remove the old email address. You must be an administrator to do this.

```

Comment:
invite a new email sounds a bit strange... Maybe we should just say invite a new user with the new email + remove the old user with the old email.

Reply:
Or "create" a new user since it's not really a new person?